src/tests/ftest/util/ior_test_base.py

```diff
@@ -78,6 +79,7 @@ def tearDown(self):
     def create_pool(self):
         """Create a TestPool object to use with ior."""
         # Get the pool params
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
This particular error is a result of pylint not being able to find `apricot/test.py` and realize that `TestWithServers.__init__()` defines `self.pool`.  With DAOS-3731 merged, this should hopefully not be a problem and this disable option should be removed.

Reply:
Will do.